storage/Utils/StorageDefines.h

```diff
@@ -133,4 +133,7 @@
 #define MKFS_UDF_BIN "/usr/sbin/mkfs.udf"
 
 
+#define UUID_REGEX "[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}"
```

Comment:
just question: storage never use uppercase in uuid? As it is generic UUID regex it can be problem if it is used on uuid that contain uppercase letters.

Reply:
I cannot say the storage never uses uppercase UUIDs but so far there is always lowercase.